src/core/variables.tf

```diff
@@ -423,3 +423,30 @@ variable "selfcare_plan_sku_capacity" {
   type        = number
   default     = 1
 }
+
+
+# legal backup storage
+variable "legalbackup_account_replication_type" {
+  type        = string
+  description = "Legal backup replication type"
+  default     = "LRS"
+}
+
+variable "legalbackup_delete_retention_days" {
+  type        = number
+  description = "Number of days to retain deleted legal backup file"
+  default     = 1
```

Comment:
with 1 blobs will be deleted after 1 day

Reply:
@pasqualedevita yes, could we set it to 14 days. This is a recycling time after an explicit deletion, right?

Reply:
no this blob retention after blob is modified

Reply:
removed by https://github.com/pagopa/io-infra/pull/84/commits/2a5244310bea68b850590847b92065c6a48ad99e